### PR TITLE
refactor(api,robot-server): Make reset opts for cal

### PR DESCRIPTION
**Pull request by sfoster1**
While we may change this with an actual UX-design inspired approach, for
now let's just add some options to make the user actually able to reset
the calibrations, somehow.

### CODE REVIEW

File: api/src/opentrons/calibration_storage/delete.py [hunk 1]
```diff
@@ -156,3 +156,14 @@ def _remove_json_files_in_directories(p: Path):
         _remove_json_files_in_directories(offset_dir)
     except FileNotFoundError:
         pass
+
+
+def delete_robot_deck_attitude():
+    """
+    Delete the robot deck attitude calibraiton.
```
Comment (b-cooper) [inline api/src/opentrons/calibration_storage/delete.py:163]:
```suggestion
    Delete the robot deck attitude calibration.
```

File: robot-server/tests/conftest.py [hunk 2]
```diff
@@ -200,3 +200,61 @@ def _get_labware_fixture(fixture_name):
...
+        "value": False
+    }
+    requests.post(url, json=data)
+    yield None
+    data['value'] = None
+    requests.post(url, json=data)
+
+
+@pytest.fixture
+def apiclient_enable_calibration_overhaul(api_client):
```
Comment (b-cooper) [inline robot-server/tests/conftest.py:236]: just for consistency's sake this should probably be `api_client_enable_calibration_overhaul`

**Review by ahiuchingau — APPROVED** [review]
All 3 resets work as intended and clear the appropriate calibrations!

**Review by b-cooper — APPROVED** [review]
Looks good 🔵  testing on robot now

File: robot-server/tests/conftest.py (hunk 2 above)
Comment (sfoster1) [reply robot-server/tests/conftest.py:236]: Ehhhhh I dunno. I wish you didn't have to have two separate fixtures for this, and I like having the target just as the first word.

Comment (b-cooper) [reply robot-server/tests/conftest.py:236]: works for me 👍

**Pull request merged**